examples/real-world/containers/RepoPage.js

```diff
@@ -73,7 +73,7 @@ RepoPage.propTypes = {
 }
 
 function mapStateToProps(state) {
-  const { login, name } = state.router.params
+  const [ login, name ] = state.routing.path.split('/').slice(1)
```

Comment:
Maybe this can be solved with something like this?

```javascript
function mapStateToProps(state, props) {
  const { login, name } = props.params
```